docs/getting-started/pip_installing.md

```diff
@@ -38,33 +38,3 @@ pip install concrete-ml
 ```
 
 This will automatically install all dependencies, notably Concrete.
-
-## Using Docker
```

Comment:
why do you remove docker?
and don't you want to link to https://community.zama.ai/t/troubleshooting-concrete-installation-on-apple-silicon/577?

Reply:
Feels weird to have that in pip installing but also would be best to keep this in the documentation somewhere.

Reply:
for the name of the file: indeed, it's not perfect, it's more for historical reasons
but if we rename or create a new file, we're going to break links, which is not great

Reply:
I removed it as it is not linked anywhere else in the doc and no longer needed. I'm not saying it's not interesting but it was more a workaround for Apple silicon than a real need, for me it would be easier to completely remove it (also less case to handle from a support point of view) but your call?

Reply:
I think we should keep it. Many people use docker as a way to avoid changing their local environment. Sometimes a virtualenv is not enough

Reply:
is it tested?

Reply:
kept the Docker part